utopia-remix/app/services/fgaService.server.ts

```diff
@@ -35,6 +35,18 @@ async function checkUserProjectPermission(
   return allowed ?? false
 }
 
+export async function getAllPermissions(projectId: string, userId: string) {
+  const { relations } = await fgaClient.listRelations({
+    user: `user:${userId}`,
+    object: `project:${projectId}`,
+    relations: fgaUserProjectPermission.map((permission) => permission),
```

Comment:
is this `map` needed?

Reply:
it was there since TS didn't allow casting from a const array into `string[]`. I changed it now to have an explicit casting using `as unknown as string[]`, to move this issue to the type world instead of the runtime :)